libs/dal/src/repositories/notification/notification.schema.ts

```diff
@@ -41,12 +41,12 @@ const notificationSchema = new Schema<NotificationDBModel>(
       type: Schema.Types.Mixed,
     },
     expireAt: Schema.Types.Date,
-    bridge: {
-      type: Schema.Types.Mixed,
-    },
```

Comment:
I have confirmed that this parameter is not used anywhere. It can safely be removed.